db/seeds/development/people.seeds.rb

```diff
@@ -4,7 +4,8 @@
   age: 27,
   email: "johanna@example.com",
   status: :established,
-  country: :se,
+  country: "SE",
```

Comment:
These were not working correctly, apparently they needed to be uppercase, which is how they're stored if you select something from the dropdown.